package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "eslint-config-digitalbazaar",
+  "version": "1.0.0",
+  "description": "basic eslint config for digital bazaar projects",
```

Comment:
I suppose this should be an actual sentence with proper capitalization etc.

Reply:
yeah agreed will replace with Digital Bazaar's eslint rules.